eng/scripts/Automation-Update-Metadata.ps1

```diff
@@ -0,0 +1,41 @@
+# Automation-Update-Metadata.ps1
+# This script is used to update metadata for the Azure SDK automation process
+
+param(
+    [Parameter(Mandatory = $true)]
+    [string]$sdkRepoPath,  # Absolute path to the root folder of the local SDK repository.
```

Comment:
I think we can just eliminate this parameter as I don't think we need it for anything given the script is already running in the context of the sdk repo.